### ISSUE TITLE
put description as hover text for medal list

### ISSUE THREAD

**Issue by Bathbot-Helper** [MaxOhn/Bathbot#428]
> could there be a more slightly more detailed option for medal list where it shows some amount of characters of the solution (or description if there is none) cutting off at the end of the first sentence if thats shorter? the current more condensed version can be tedious sometimes as u have to click each hyperlink to get an idea of how they work

[Original message by @cd36](https://discord.com/channels/741040473476694159/741047663532834827/1121597559484469379)

**Comment by surrenderoz** [comment]
which file or where to put description as hover model?

**Comment by MaxOhn** [comment]
> which file or where to put description as hover model?

https://github.com/MaxOhn/Bathbot/blob/8a1906ddb54968c857319d8c2dc5c7c1ed89d52b/bathbot/src/active/impls/medals/list.rs#L49-L50

**Comment by surrenderoz** [comment]
Should I use this bot in discord to understand the requirements.
I'm not sure how the model will display

**Comment by MaxOhn** [comment]
Yea it would be best if you go through the setup steps in the [readme](https://github.com/MaxOhn/Bathbot/blob/main/README.md#setup), apply some changes to the code, and then test it by running on your own discord bot token.

It's not the simplest project to setup for on the first time 😅

**Comment by surrenderoz** [comment]
yeah  I know it is not the simple for first time but I'll give it a try
